cdap-ui/app/cdap/api/__mocks__/preference.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
```

Comment:
Not sure if the mocks file would be useful unless we have unit tests for the same. Would be awesome if we can have unit tests for preferences component